Update YAML docs to support idp_image_url (#26104)
fleetdm/fleet: docs/Configuration/yaml-files.md

- `enable_sso` (default: `false`)
- `idp_name` is the human-friendly name for the identity provider that will provide single sign-on authentication (default: `""`).

> - `idp_image_url` is a link to a logo or other image that is displayed at login (default: `""`).

Reviewer: Keeping the language the same as the REST API docs:
```suggestion
- `idp_image_url` is an optional link to an image such as a logo for the identity provider. (default: `""`).
```
